packages/guui/components/Grid/index.js

```diff
@@ -114,13 +113,19 @@ export const Row = ({
     htmlTag,
     children,
 }: {
-    htmlTag?: string,
-    children: React$Node,
-}) => {
-    const GridRow = RowStyled.withComponent(htmlTag);
-
-    return <GridRow>{children}</GridRow>;
-};
+    htmlTag: string,
+    children: React.Node,
+}) => (
+    <>
```

Comment:
What is this magic?!

Reply:
[Fragment](https://reactjs.org/docs/fragments.html)

I'm not 100% sure this is a legit use of fragments, but I couldn't think of another way of defining a component with a dynamic tag.